### PR TITLE
Address function and variable naming lint errors

### PR DESCRIPTION
**Pull request by trshafer**
## Proposed Changes

  * Internal refactor

**Release Note**

```release-note
NONE
```

### CODE REVIEW

File: pkg/controller/sinks/sinks_test.go [hunk 1]
```diff
@@ -151,23 +151,7 @@ func getAddressable_noStatus() *unstructured.Unstructured {
 	}
 }
 
-func getAddressable_nilAddress() *unstructured.Unstructured {
```
Comment (evankanderson) [inline pkg/controller/sinks/sinks_test.go]: Was this not used?

Comment (evankanderson) [reply pkg/controller/sinks/sinks_test.go]: If so, does that mean we are missing a test?

File: pkg/reconciler/containersource/containersource_test.go [hunk 2]
```diff
@@ -637,7 +637,7 @@ func TestObjectHasDeleteTimestamp(t *testing.T) {
 	if diff := cmp.Diff(want, got); diff != "" {
 		t.Errorf("unexpected returned object (-want, +got) = %v", diff)
 	}
-	var wantErr error = nil
+	var wantErr error
 	if diff := cmp.Diff(wantErr, gotErr); diff != "" {
```
Comment (evankanderson) [inline pkg/reconciler/containersource/containersource_test.go]: Why does this not just check that `gotErr == nil`?

File: pkg/controller/sinks/sinks_test.go (hunk 1 above)
Comment (trshafer) [reply pkg/controller/sinks/sinks_test.go]: The function was not called. Here's what that test would look like: https://github.com/knative/eventing-sources/pull/316. I can wait for that PR to be merged in and then fix the name change in this PR.

**Comment by evankanderson** [comment]
(Otherwise, this removal will break #316 )

FWIW, I'd have just added the #316 change to this PR.

File: pkg/reconciler/containersource/containersource_test.go (hunk 2 above)
Comment (trshafer) [reply pkg/reconciler/containersource/containersource_test.go]: Done

**Pull request merged**